### PR TITLE
[GRIFFIN-194] single call to fetch all tables of all dbs

### PR DESCRIPTION
**Pull request by chemikadze**
Significantly improves opening speed for measure dialogs.
Previously taking 10+ seconds on huge schemas, with this call
it takes fraction of second to load.

### CODE REVIEW

File: service/src/main/java/org/apache/griffin/core/metastore/hive/HiveMetaStoreServiceImpl.java [hunk 1]
```diff
@@ -99,8 +100,17 @@ public HiveMetaStoreServiceImpl() {
     public List<Table> getAllTable(String db) {
         return getTables(db);
     }
-
-
+    
+    @Override
+    @Cacheable(unless = "#result==null || #result.isEmpty()")
```
Comment (guoyuepeng) [inline service/src/main/java/org/apache/griffin/core/metastore/hive/HiveMetaStoreServiceImpl.java]: ok, add one more service for all table names.

File: ui/angular/src/app/measure/create-measure/ac/ac.component.ts [hunk 2]
```diff
@@ -644,48 +644,37 @@ export class AcComponent implements OnInit, AfterViewChecked {
   }
 
   ngOnInit() {
-    let allDatabases = this.serviceService.config.uri.dblist;
-    let getTableNames = this.serviceService.config.uri.tablenames;
+    let getTableNames = this.serviceService.config.uri.dbtablenames;
```
Comment (guoyuepeng) [inline ui/angular/src/app/measure/create-measure/ac/ac.component.ts]: add one more endpoint.

**Comment by chemikadze** [comment]
@guoyuepeng can you please clarify your comments, please? Not very clear what changes are required, if any.

**Comment by guoyuepeng** [comment]
@chemikadze  LGTM. 
Just rushed out for my meeting. sorry for confusing.

**Pull request closed without merging**